src/Jackett.Common/Indexers/TorrentDay.cs

```diff
@@ -170,20 +170,18 @@ public override async Task<IndexerConfigurationStatus> ApplyConfiguration(JToken
             var result = await RequestLoginAndFollowRedirect(LoginUrl, pairs, configData.CookieHeader.Value, true, null, LoginUrl);
             await ConfigureIfOK(result.Cookies, result.Content != null && result.Content.Contains("logout.php"), () =>
             {
-                CQ dom = result.Content;
-                var messageEl = dom["#login"];
-                messageEl.Children("form").Remove();
-                var errorMessage = messageEl.Text().Trim();
+                var parser = new HtmlParser();
+                var dom = parser.ParseDocument(result.Content);
+                var messageEl = dom.QuerySelector("#login");
+                foreach (var child in messageEl.QuerySelectorAll("form"))
+                    child.Remove();
```

Comment:
We should probably add this to a global extension method to make it less error prone for future use.